Sources/Sentry/include/SentryClient.h

```diff
@@ -45,77 +47,20 @@ SENTRY_NO_INIT
 
 @property(nonatomic, strong) SentryOptions *options;
 
-/**
- * This property will be filled before the event is sent.
- */
-@property(nonatomic, copy) NSString *_Nullable releaseName;
 
 /**
- * This property will be filled before the event is sent.
+ * This block can be used to modify the event before it will be serialized and sent
  */
-@property(nonatomic, copy) NSString *_Nullable dist;
+@property(nonatomic, copy) SentryBeforeSerializeEvent _Nullable beforeSerializeEvent;
```

Comment:
Note, this should be a property on the Client anymore, instead this should become the `beforeSend` callback in the options, we can do this in another PR.

Reply:
done